src/Components/AjaxSelect/Entities/AbstractEntity.php

```diff
@@ -69,9 +74,13 @@ public function areValidValues(array $values) {
 		$query = $this->createQueryObject()
 			->byId($values);
 
-		$this->filterQueryObject($query);
+		if (method_exists($this, 'filterQueryObject')) {
```

Comment:
Proč tohle není abstraktní funkce a nebo teda prázdná protected?

Reply:
no protoze pak nemuzes definovat tridu u parametru... a musis to delat pres anotaci... tezko rict co je lepsi...